python/tutorials/09-persistent-matmul.py

```diff
@@ -77,7 +84,10 @@ def matmul_kernel(a_ptr, b_ptr, c_ptr,  #
         a_ptrs += BLOCK_SIZE_K * stride_ak
         b_ptrs += BLOCK_SIZE_K * stride_bk
 
-    c = accumulator.to(tl.float8e4nv)
+    if FP8_OUTPUT:
```

Comment:
nit: can we rely on c_ptr type or pass directly the dtype?

Reply:
Good point!